src/pages/en/blog/2023-05-19-qm-needs-you.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- MORE -->
+
+#### What's holding the team back?
+
+The Quilt Mappings team is one of the largest in Quilt, but we’re still struggling to keep up. Why? There are a few reasons. Most of the team is simply busy and doesn’t have the time to submit new mappings, and those of us that do have time have other exciting projects to work on! We’re building an entirely new mappings toolchain, always matching to the latest snapshots, and delivering huge improvements to Enigma, the program that we use to create mappings. That’s not to say we’re not working on mappings all the time, though, and we’re always here to deliver quick reviews on pull requests!
+
+#### Why even bother?
+
+Quilt Mappings is different from the other two sets of maintained mappings on the market: Mojmap and Yarn. We check our names against both sets constantly to make sure that we’re delivering the most accurate ones possible. Mojang names are often unintuitive, and aren’t built for modders: they’re made for the developers working on the game itself. Contrary to Mojmap, Yarn aims to deliver names that are easy to understand for modders, but since their names are not checked against Mojang’s intentions, they can sometimes be confusing and contradictory to the intended usage of the code. Both sets are excellent on their own, and Quilt Mappings is made to be a hybrid of the two, shipping clear, concise, and accurate names inspired by both. We’ve also implemented extensive checking for typos, Britishisms and convention violations to make sure no problematic mappings are published. Despite their drawbacks, Yarn and Mojang have 99% and 100% completion of class names respectively, and we can’t truly call ourselves the best with our measly 97.
+
+## How can I help?
+
+The biggest way you can help with QM is by contributing new mappings to the large swaths of the game that are missing names. All you need to know is English and have some basic knowledge of reading code. We have a couple cute documents already written up on how to contribute: check out our [CONTRIBUTING.md](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre2/CONTRIBUTING.md) to find out everything you need to know before mapping! Making new mappings isn’t the only way to help, and if you want to improve our existing names, [our issue tracker](https://github.com/QuiltMC/quilt-mappings/issues) is a great place to find suggestions of names to change. However, be warned that if you change names made by me you run the risk of incurring my wrath (affectionate). Opening Enigma will immediately give you an idea of what needs doing in terms of new mappings, thanks to our handy little status icons next to each class. If a class has a check mark, you can move on, but anything with a yellow dot or red x needs your help! This adorable graph, generated by the Mapping Stats feature in Enigma, can help show you where to start.
```

Comment:
> All you need to know is English

Something feels off about this. The CONTRIBUTING.md reference should also be updated to the newest branch after Ennui's FCP length adjustments are merged. A link to `good first issue`s would also be nice.

We could also mention here that if someone has a question about how to use Enigma, what to name some method, or any other question, they are welcome to ask in the Discord

Reply:
I can just remove that bit I think, if you're reading this you already know english